### DISCUSSION TITLE
Bootstrap compatibility

### DISCUSSION THREAD

**Discussion by synmux** [dracula/dracula-ui#68] [Ideas] [answered]
Is Bootstrap 5 compatibility planned or desired? It seems a lot of the components operate similarly, and it would be great to be able to load Dracula on top of Bootstrap and benefit from the heavy lifting (and tooling) which exists for Bootstrap.

**Comment by helderberto** [comment, marked as answer]
I think you can load Dracula and some parts of Bootstrap too. I just don't know if it will have an override of features from Dracula with the Bootstrap usage like CSS resets and defaults from each one. I recommend you create a simple POC to try it.

**Reply by nettofarah** [reply]
Yeah, agreed.
There's no reason why Dracula UI shouldn't work with any existing CSS frameworks like Bootstrap or Tailwind.

Our CSS class names are namespaced with `.drac` in order to preserve compatibility with existing websites.

**Reply by GitToDaChoppa** [reply]
Used it with Bulma today (actually, Bulmaswatch CDN, which is even harder because I was unable to edit). 

It worked great, only had one or two little issues... all of which were fixed in a style-tag because I didn't want to add a css file. I think it was the link hovers that gave me an issue... the other thing was minor or just my own customization.

**Reply by ArjaanBuijk** [reply]
I can confirm that it works  to use Dracula with Bootstrap 5.
